configs/machines/aleph.yaml

```diff
@@ -46,53 +46,127 @@ hyper_flag: ""
+                        - "load fftw/2.1.5.9"
+                        - "load nco/4.9.4"
+                        - "load proj4/5.1.0"
+                        - "load python/3.9.1"
+                        - "list"
+                add_export_vars:
+                        CMAKE_C_COMPILER: /home/awiiccp2/software/HPC_libraries/intel-18.0.3.222_ompi_20230717/bin/mpicc
+                        CMAKE_CXX_COMPILER: /home/awiiccp2/software/HPC_libraries/intel-18.0.3.222_ompi_20230717/bin/mpicxx
+                        IO_LIB_ROOT: /home/awiiccp2/software/HPC_libraries/intel-18.0.3.222_ompi_20230717
+                        LD_LIBRARY_PATH: $IO_LIB_ROOT/lib:$LD_LIBRARY_PATH
+
+                        SZIPROOT: $IO_LIB_ROOT
+                        HDF5ROOT: $IO_LIB_ROOT
+                        HDF5_ROOT: $HDF5ROOT
+                        NETCDFROOT: $IO_LIB_ROOT
+                        NETCDFFROOT: $IO_LIB_ROOT
+                        ECCODESROOT: $IO_LIB_ROOT
+
+                        HDF5_C_INCLUDE_DIRECTORIES: $HDF5_ROOT/include
+                        NETCDF_Fortran_INCLUDE_DIRECTORIES: $NETCDFFROOT/include
+                        NETCDF_C_INCLUDE_DIRECTORIES: $NETCDFROOT/include
+                        NETCDF_CXX_INCLUDE_DIRECTORIES: $NETCDFROOT/include
+                        OASIS3MCT_FC_LIB: '"-L$NETCDFFROOT/lib -lnetcdff"'
+                        PATH: $IO_LIB_ROOT/bin:$PATH
+                        MPIFC: ftn
+
+        intel_mpich:
+                fc: ftn
+                f77: ftn
+                mpifc: ftn
+                mpicc: cc
+                cc: cc
+                cxx: CC
+                module_actions:
+                        - "unload PrgEnv-cray"
+                        - "load PrgEnv-intel/6.0.5"
+                        - "load craype-x86-skylake"
+                        - "load cray-mpich/7.7.3"
+                        - "load gcc/7.3.0"
+                        - "load pbs"
+                        - "load cmake/3.14.0"
+                        - "load cdo/1.9.5"
+                        - "load fftw/2.1.5.9"
+                        - "load nco/4.9.4"
+                        - "load proj4/5.1.0"
+                        - "load python/3.9.1"
+                        - "load cray-netcdf-hdf5parallel/4.6.1.3"
+                        - "load cray-hdf5-parallel/1.10.2.0"
+                        - "list"
+                add_export_vars:
+                        ECCODESROOT: /home/awiiccp2/software/HPC_libraries/eccodes_intel_mpich/
+
+                        HDF5_C_INCLUDE_DIRECTORIES: $HDF5_ROOT/include
+                        NETCDF_Fortran_INCLUDE_DIRECTORIES: $NETCDFFROOT/include
+                        NETCDF_C_INCLUDE_DIRECTORIES: $NETCDFROOT/include
+                        NETCDF_CXX_INCLUDE_DIRECTORIES: $NETCDFROOT/include
+                        OASIS3MCT_FC_LIB: '"-L$NETCDFFROOT/lib -lnetcdff"'
+                        PATH: $IO_LIB_ROOT/bin:$PATH
+                        MPIFC: ftn
```

Comment:
I would recommend to add a warning here saying that it runs but it doesn't compile and who to contact :)

https://esm-tools.readthedocs.io/en/latest/yaml.html#error-handling-and-warning-syntax

Reply:
that's really cool!

Reply:
I added warnings that it compiles but doesnt run. Did I do it right?

Reply:
Almost, see suggestion above

Reply:
ty!